cmd/kops/create_cluster.go

```diff
@@ -359,6 +365,22 @@ func RunCreateCluster(f *util.Factory, cmd *cobra.Command, args []string, out io
 		}
 	}
 
+
+	// Network Topology
+	switch  c.Topology{
```

Comment:
There are notes in the struct definition, in the docs, in the cobra commands, etc :)